src/toil/test/mesos/mesosTest.py

```diff
@@ -1,70 +0,0 @@
-# Copyright (C) 2015-2016 Regents of the University of California
```

Comment:
@jeffrey856 Why did you decide to delete this test? What makes it redundant? Is Mesos sufficiently tested by some other tests?

Reply:
I assumed the tests from the batchstytemTest.py under test would run a similar kind of job since it does run setup and creates a cluster, teardown, and a test within MesosBatchSystemTest() and or MesosBatchSystemJobTest() classes.

Reply:
OK, that makes sense.